### DISCUSSION TITLE
Notifications marked as 'read' although they are unread

### DISCUSSION THREAD

**Discussion by ghost** [community/community#38500] [Other Feature Feedback, Questions, & Ideas]
I am not sure why all my notifications are marked as 'read' today even though they are unread.

**Comment by david-engelmann** [comment]
[You can configure your notification settings from this url](https://github.com/settings/notifications). Depending on how it's setup, somethings will not come through as notifications through github, instead only coming through email. Check out how my notifications are setup.

![image](https://user-images.githubusercontent.com/81196676/200395549-953ffae8-4e8a-4e6c-8573-d0f3c9d0c6b3.png)

Having most things set to email instead of "on Github", I receive very few notifications

**Reply by ghost** [reply]
I have 'email and github' for everything. It was working fine until yesterday. Let me share screenshots:

![image](https://user-images.githubusercontent.com/94559964/200396109-9536594c-10c7-4f76-8d7f-f7fe95276c57.png)

I have not read last comment in all these pull requests still they are marked 'read'.

![image](https://user-images.githubusercontent.com/94559964/200396362-4711ca6c-54dc-4f5d-921f-e5937f76a4ff.png)

Old notifications in which 'unread' works as expected.

**Reply by david-engelmann** [reply]
This feels like a legit bug, It doesn't seem like anything in those specific pr's would make them different from the ones behaving correctly!

**Reply by Mrsingh8855** [reply]
Ok 👍

**Reply by DmitriyMV** [reply]
Same issue happening to me.

**Comment by airtower-luna** [comment]
Does your mail client load external resources in HTML mails? I remember reading about other people who had a similar problem, and found that there was a tracking pixel in the HTML version of GitHub notifications, which when loaded marked the notification as read. Looking at the source of one of my notification mails I see something like this:

```
<img src="https://github.com/notifications/beacon/[...].gif height="1" width="1" alt="" />
```

Really bad design, but it explains the problem. :sweat_smile: 

Turning off loading external resources in HTML mails is good practice for security and privacy anyway, because spammers and phishers might use similar tricks.

**Reply by tdtm** [reply]
Thanks for the heads up. That's kinda clever and I see why they'd do that, but being unaware of this behaviour makes it likely to miss things! I wish this behaviour could be configurable.

**Reply by 2called-chaos** [reply]
That is really bad design since providers such as gmail request/proxy these on arrival specifically for the security and privacy aspect. Ever since the UI update I'm missing things left right and center because my notifications are virtually broken

**Comment by nschonni** [comment]
I'm seeing this crop up again today

**Comment by 2called-chaos** [comment]
I think ever since the UI update ages ago my notifications are broken. It happens even without opening emails but providers may prefetch the beacons. At least gmail does so for privacy reasons (anti-tracking). If it is the mails even, idk but I haven't had an unread notification in ages and I miss a lot of things
